docs/monitoring/introduction.rst

```diff
@@ -0,0 +1,17 @@
+Monitoring within Fledge is a varied subject and has different meanings to the different user personas that are using Fledge.
+
+Monitoring can take the form of
+
+  - Monitoring of the state of the various services that are run to implement the overall Fledge functionality.
+
+  - Monitoring the resources that Fledge needs to operation.
+
+  - Monitoring the flow of data within the various pipelines implemented by a Fledge instance.
+
+  - Monitoring for configuration changes of the pipelines that operate on the data.
+
+  - Monitoring the quality of the data that is flowing in the Fledge pipelines.
+
+  - And finally, the one that is the major purpose of Fledge, monitoring the state of the machines and processes connected to the Fledge instance.
```

Comment:
"...major purpose of Fledge, monitoring the state..." -> "...major purpose of Fledge: monitoring the state..."